src/screens/MainScreen/index.js

```diff
@@ -21,26 +23,38 @@ const MainScreen = () => {
     getCalls();
   }, [currentUser]);
 
+  const joinNextSession = async () => {
+    await subscribeNextSession(currentUser.id, nextSession);
+    alert("You're now subscribed to the next session of Corona is Blind")
+  }
+
   const reset = async () => {
     await resetFlake(currentUser.id);
     alert("Done! We're glad you're still interested!");
   };
 
   return (
     <Container>
-      {currentUser ? (
+      {currentUser && currentSession ? (
         <>
           {currentUser.flake &&
             <Banner
               onClick={() => reset()}
               text="We've noticed you missed attending some of your calls. If you still wish to participate in this session of Corona is Blind, click on this banner."
             ></Banner>
           }
+          {
+            currentSession.done && currentUser.session !== nextSession &&
+            <Banner
+              onClick={() => joinNextSession()}
+              text={`The current session ends on ${new Date(currentSession.endDate).toLocaleDateString("en-US")}. If you wish to participate in the next session, click on this banner.`}
```

Comment:
I don't think you need to create a new Date object for it because `currentSession.endDate` should already be a Date object? I might be wrong though

Reply:
I made it like this to handle in case it was input as a timestamp. Whether it is input as a date or a timestamp, it will return a date.

Reply:
Makes sense